groups/xml-infrastructure/rds.tf

```diff
@@ -60,6 +60,12 @@ module "xml_rds" {
   # Enhanced Monitoring
   monitoring_interval = "30"
   monitoring_role_arn = data.aws_iam_role.rds_enhanced_monitoring.arn
+  enabled_cloudwatch_logs_exports = [
+    "alert",
+    "audit",
+    "listener",
+    "trace"
```

Comment:
Should be var from profiles? we probably want to control log levels per environment

Reply:
The above was set in Dev, I would expect that Live should be same or more compared to lower environments at least in the case of logging?